packages/@react-types/tag/package.json

```
@@ -10,7 +10,9 @@
   },
   "dependencies": {
     "@react-stately/grid": "^3.4.1",
-    "@react-types/shared": "^3.16.0"
+    "@react-types/grid": "^3.1.5",
+    "@react-types/shared": "^3.16.0",
+    "react-stately": "^3.19.0"
```

```suggestion
    "@react-types/shared": "^3.16.0",
    "react-stately": "^3.19.0"
```
Also maybe the IDE autofilled the the `react-stately` here, prob can point directly to `react-stately/list`